DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
```

Comment:
Still too much props. RootProjetDir cannot be defined only once ? and import props in csproj ?

Reply:
Needed for relative path in each Common property in PropertyGroup